### PR TITLE
Restored OOMTest, partially, so it does run. 

### PR DESCRIPTION
**Pull request by vpatryshev**
May help with false error reports, let's see.

### CODE REVIEW

File: h2o-core/src/test/java/water/OOMTest.java [hunk 1]
```diff
@@ -9,11 +9,12 @@
...
 
-@Ignore
 public class OOMTest extends TestUtil {
   @BeforeClass() public static void setup() { stall_till_cloudsize(1); }
 
+  // TODO(vlad): restore the commented out checks; they may mean something
   @Test public void testClean() throws InterruptedException {
+    Scope.enter();
     final int log_rows_per_chk = 6;
     final int nchks = 1024/(1<<log_rows_per_chk); // 1024/(1<<4) = 64 chunks
```
Comment (st-pasha) [inline h2o-core/src/test/java/water/OOMTest.java]: isn't `x / (1 << y)` same as `x >> y` ?

**Comment by vpatryshev** [comment]
@h2o-ops please test!

File: h2o-core/src/test/java/water/OOMTest.java (hunk 1 above)
Comment (vpatryshev) [inline h2o-core/src/test/java/water/OOMTest.java]: :) Have to check extreme cases; probably.

**Comment by abal5** [comment]
@h2o-ops please test!

**Pull request closed without merging**